src/Core/Manifest/ClassManifestErrorHandler.php

```diff
@@ -0,0 +1,33 @@
+<?php declare(strict_types=1);
```

Comment:
This is PHP 7 only? I believe this'll throw a warning in 5.6.

Reply:
Derp derp, damn you copy paste. Fixed.